.github/workflows/ci.yaml

```diff
@@ -26,6 +26,8 @@ jobs:
           components: rustfmt
       - uses: r7kamura/rust-problem-matchers@v1
       - run: cargo +${{ matrix.rust }} fmt --all -- --check
+      - run: cargo +${{ matrix.rust }} fmt --all -- --check
```

Comment:
Note: I couldn't see a good way to specify the working directory for `cargo deny`.  I'm also thinking we'll need to copy the `deny.toml` or soft link it into the `panic/abort` dir.  

During development of the panic-logger I ran into, https://github.com/rust-lang/cargo/issues/6745, which prohibits adding `panic` to the workspace and excluding `panic/abort`, because of that I think we'll need to come up with a longer term plan of where to house `panic/abort` so I punted on deny for now.

Reply:
created #27 to not lose track of this